provisioning/roles/jenkins/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Ensure dependencies are installed.
+  apt: pkg=curl state=installed
```

Comment:
Curl is already installed with common role, along with other tools https://github.com/netgen/summercamp-2015/blob/master/provisioning/roles/common/vars/main.yml#L10